docs/config/development/feature-from-scratch/index.rst

```diff
@@ -0,0 +1,385 @@
+===============================
+Creating a Feature from Scratch
+===============================
+
+This guide will outline the process of creating a new feature from the beginning.
+
+If you haven't already, please read the
+:doc:`Config Development Introduction </config/development/introduction>` &
+:doc:`Creating A Pack From Scratch </config/development/pack-from-scratch/index>`
+for more information before continuing.
+
+For a more detailed and in-depth guide about creating a new feature from scratch, please read this unofficial
+development guide, `Feature Config <https://terra.atr.sh/#/page/feature%20config>`__.
+
+Setting up a New Feature
+========================
+
+`PROCEDURE`
+
+1. Add feature stage(s)
+-----------------------
+
+.. card::
+
+    Your pack will need feature stages in order to generate features.
```

Comment:
Perhaps something more explanatory like 'generation is divided up into generation stages, in order to generate any features your pack will need to define at least one feature stage'